content/en/docs/concepts/services-networking/service.md

```diff
@@ -188,9 +188,10 @@ selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
 ### Over Capacity Endpoints
-If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.21 (or later)
-cluster annotates that Endpoints with `endpoints.kubernetes.io/over-capacity: warning`.
-This annotation indicates that the affected Endpoints object is over capacity.
+If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.22 (or later)
+cluster annotates that Endpoints with `endpoints.kubernetes.io/over-capacity: truncated`.
```

Comment:
nit: Can you be more specific? What annotates the Endpoints resource?

Reply:
Added a small addition stating that the endpoint controller truncates the resource. Please let me know if you think we should expand more. I am not sure how detailed I should be here.

Reply:
“the control plane has truncated” would be fine.